examples/epd_ept_ts.py

```diff
@@ -0,0 +1,71 @@
+"""
+===============================================
+Finding and Plotting Solar Orbiter EPD/EPT Data
+===============================================
+
+This example demonstrates how to search for, download, and visualize Level-3
+1-minute averaged data of the Electron Proton Telescope (EPT) of Solar Orbiter's
+Energetic Particle Detector (EPD) from the SOAR archive, load it into a SunPy
+TimeSeries, and plot it.
+"""
+
+import matplotlib.pyplot as plt
+from sunpy.net import Fido, attrs as a
+import sunpy_soar
+from sunpy.timeseries import TimeSeries as ts
+
+###############################################################################
+#
+# By importing `sunpy_soar`, the Solar Orbiter Archive (SOAR) is **automatically**
+# registered as a data provider in `Fido`. This allows us to directly query and 
+# download Solar Orbiter data just like any other SunPy Fido search.
+#
+# Now, we will use `Fido` to search for EPT Level-3 data.
+# We will request 1-minute averaged data over a two-day period.
+
+search_results = Fido.search(
+    a.Time("2022-09-04", "2022-09-06 00:00"),
+    a.Instrument.epd,
+    a.soar.Product('epd-ept-1min'),
+    a.Level(3)  
+)
+
+print(search_results)
+
+###############################################################################
+# The search results contain a list of available files matching our query.
+# We use `Fido.fetch` to download the files.
+
+downloaded_files = Fido.fetch(search_results)
+
+###############################################################################
+# Loading the EPD/EPT Data into a SunPy TimeSeries
+# ------------------------------------------------
+#
+# The downloaded file contains time-series of energetic charged particle measurements.
+# We use `sunpy.timeseries.TimeSeries` to load and analyse the data.
+#
+# The **concatenate=True** argument ensures that if multiple files are downloaded,
+# they are combined into a single continuous TimeSeries object.
+
+ept_ts = ts(downloaded_files, concatenate=True)
```

Comment:
Just a small style thing but it a little bit weird to see a class as lower case and is it really needed?
```suggestion
ept_ts = TimeSeries(downloaded_files, concatenate=True)
```

Reply:
True, there is no real reason for this notation. I just did copy some old code example that looked like this. I now adjusted the code accordingly.